src/pharmpy/workflows/model_database/local_directory.py

```diff
@@ -30,14 +30,24 @@
 FILE_LOCK = '.lock'
 
 
-def get_modelfit_results(model, path):
+def get_modelfit_results(model, path, esttool=None):
     # FIXME: This is a workaround. The proper solution is to only read the results.json from
     # the database. For this to work roundtrip of DataFrames in json is needed.
     # This is currently broken because of rounding issue in pandas
     # Also the modelfit_results attribute will soon be removed from model objects.
     import pharmpy.model.external.nonmem as nonmem_model
     import pharmpy.tools.external.nonmem as nonmem
 
+    if esttool is not None:
+        if esttool == 'dummy':
+            import pharmpy.tools.external.dummy as tool
+        elif esttool == 'nonmem':
+            import pharmpy.tools.external.nonmem as tool
+        elif esttool == 'nlmixr':
+            import pharmpy.tools.external.nlmixr as tool
+        res = tool.parse_modelfit_results(model, path)
+        return res
```

Comment:
Is this necessary? Given that there is `esttool` as an option to the top level `tools.external.parse_modelfit_results` function.